mac/22/_bugsnag_x86_64.crash

```diff
@@ -1,27 +1,25 @@
-Segmentation fault: 11: 
-
-bugsnag/BugsnagEvent.m:91:0 -[BugsnagEvent getStackTraceWithException:]
-bugsnag/BugsnagEvent.m:50:0 -[BugsnagEvent addSignal:]
-bugsnag/BugsnagNotifier.m:73:0 -[BugsnagNotifier notifySignal:]
-bugsnag/Bugsnag.m:67:0 handle_signal
-libsystem_platform.dylib:0 sigtramp
-<span class="cp-wrong">Missing frame that shows where the crash occured</span>
-CrashProbe/CRLMainWindowController.m:72:0 -[CRLMainWindowController causeCrash:]
-libsystem_trace.dylib:0 os_activity_initiate
-AppKit:0 -[NSApplication sendAction:to:from:]
-AppKit:0 -[NSControl sendAction:to:]
-AppKit:0 _26-[NSCell _sendActionFrom:]_block_invoke
-libsystem_trace.dylib:0 os_activity_initiate
-AppKit:0 -[NSCell _sendActionFrom:]
-AppKit:0 -[NSButtonCell _sendActionFrom:]
-libsystem_trace.dylib:0 os_activity_initiate
-AppKit:0 -[NSCell trackMouse:inRect:ofView:untilMouseUp:]
-AppKit:0 -[NSButtonCell trackMouse:inRect:ofView:untilMouseUp:]
-AppKit:0 -[NSControl mouseDown:]
-AppKit:0 -[NSWindow _reallySendEvent:]
-AppKit:0 -[NSWindow sendEvent:]
-AppKit:0 -[NSApplication sendEvent:]
-AppKit:0 -[NSApplication run]
-AppKit:0 NSApplicationMain
-CrashProbe/main.m:13:0 main
-libdyld.dylib:0 start
+Exception Type:  EXC_BAD_ACCESS (SIGBUS)
+Exception Subtype: KERN_NO_ACCESS
+Attempted to dereference garbage pointer 0x8.
+                                                                                                   
+0  CrashProbe            crash (CRLCrashSwift.swift:36)                                        
```

Comment:
Same as for iOS. The symbol doesn't look complete. I would mark this as incomplete.